ajax/libs/rambda/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "rambda",
+  "filename": "webVersion.js",
+  "version": "0.5.1",
+  "description": "Lightweight alternative to Ramda",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/selfrefactor/rambda.git"
+  },
+  "keywords": [
+    "ramda",
+    "fp",
+    "functional",
+    "utility"
+  ],
+  "author": {
+    "name": "self_refactor",
+    "url": "https://github.com/selfrefactor"
+  },
+  "license": "MIT",
```

Comment:
@pvnr0082t The author changed to `MIT` in `package.json` https://github.com/selfrefactor/rambda/blob/master/package.json#L22
Please confirm.